apps/api/Dockerfile

```diff
@@ -20,6 +20,8 @@ RUN npm install --omit dev
 FROM node:16-alpine
 WORKDIR /app
 
+RUN apk add --no-cache krb5 krb5-libs
```

Comment:
whats for ?

Reply:
this is for Kerberos support inside image. Need it to authenticate on shared resources.

Reply:
which ressources ? I don't think its a suitable use case to have to add that directly to the base docker image.

Reply:
It is needed for Kerberos authentication while mounting NFS shared folder to save log files into it.